cardname/lib/cardname/manipulate.rb

```diff
@@ -36,6 +36,11 @@ def num_parts
       parts.length
     end
 
+    def prepend_joint
+      joint = self.class.joint
+      self =~ /^#{Regexp.escape joint}/ ? self : (joint + self)
+    end
```

Comment:
Henry actually pinged me recently to ask about using different joints.  It would probably be good to look for easy opportunities to change `+` to `Card::Name.joint`.